server/src/test/java/org/elasticsearch/cluster/routing/allocation/allocator/DesiredBalanceComputerTests.java

```diff
@@ -595,6 +603,7 @@ public void testDesiredBalanceShouldConvergeInABigCluster() {
             var replicas = randomIntBetween(1, nodes - 1);
             totalShards += shards * (replicas + 1);
             var inSyncIds = randomList(shards * (replicas + 1), shards * (replicas + 1), () -> UUIDs.randomBase64UUID(random()));
+            var shardSize = randomLongBetween(1_000L, 1_000_000L);
```

Comment:
This is in bytes right? Seems kinda small if so? We should have some shards in the MiBs range (but not kiB really) and many in the 10s-of-GiBs range.

Reply:
Correct, I updated that, however I do not think this is very important as node size is determined based on sum of shard sizes.